lib/msal-browser/test/cache/BrowserCacheManager.spec.ts

```diff
@@ -326,24 +326,34 @@ describe("BrowserCacheManager tests", () => {
 
                     expect(
                         browserSessionStorage.getAccount(
-                            testAccount.generateAccountKey()
+                            AccountEntityUtils.generateAccountKey(testAccount)
                         )
                     ).toEqual(testAccount);
                     expect(
-                        browserSessionStorage.getAccount(
-                            testAccount.generateAccountKey()
+                        AccountEntityUtils.isAccountEntity(
+                            //@ts-ignore
```

Comment:
What does this show if it is not ts-ignored?

Reply:
It complains because `getAccount()` can return null and `isAccountEntity()` won't take in null as an input